unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/HelpAndSupportHUD/HelpAndSupportHUDController.cs

```diff
@@ -12,9 +13,12 @@ public class HelpAndSupportHUDController : IHUD
         internal const string FAQ_URL = "https://docs.decentraland.org/decentraland/faq/";
         public IHelpAndSupportHUDView view {  get; }
 
-        public HelpAndSupportHUDController(IHelpAndSupportHUDView view)
+        private ISupportAnalytics supportAnalytics;
```

Comment:
variable could be just `analytics` as well

Reply:
we could have specific analytics of other things so I'd rather keep it like this

Reply:
I would suggest to follow YAGNI principle and change the name when it is required (with opposite to _"could be required"_). However, it is minor here, so you can keep it also like that